src/syft/core/pointer/pointer.py

```diff
@@ -13,12 +46,35 @@
 
 
 class Pointer(AbstractPointer):
+    """
+    Pointer is the handler when interacting with remote data.
+
+    Automatically generated subclasses of Pointer need to be able to look up
```

Comment:
... "for more information on how subclasses of pointer are automatically generated..., please see <ast module discussing automatic generation... probably st.klass>"

This notion that "this Pointer class is never directly use but is subclassed" should probably be added to the story at the top.